src/frameworks/drupal8/drush.md

```diff
@@ -62,6 +50,24 @@ Add a new file named `.environment` to the root of your your project's git repos
 export PATH=/app/vendor/bin:$PATH
```

Comment:
Let's use the same code from the example that lets the use of the same `.environment` file in PE: https://github.com/platformsh/platformsh-example-drupal8/blob/master/.environment#L8-L11